bin/fillTestData/Tribes.js

```diff
@@ -36,66 +55,90 @@ var tribeImageUUIDs = [
   '69a500a4-a16e-4c4d-9981-84fbe310d531'
 ];
 
-var addTribes = function (max) {
+var addTribes = function () {
   var index = 0;
+  var max = argv._[0];
+  var verbose = (argv.verbose === true);
+  var limit = (argv.limit === true);
 
   // Add tribes
   console.log('Generating ' + max + ' tribes...');
   if (max > 2000) {
     console.log('...this might really take a while... go grab some coffee!');
   }
 
-  console.log(chalk.white('--'));
-  console.log(chalk.green('Trustroots test tribes data'));
-  console.log(chalk.white('--'));
+  if (verbose) {
+    console.log(chalk.white('--'));
+    console.log(chalk.green('Trustroots test tribes data'));
+    console.log(chalk.white('--'));
+  }
```

Comment:
I feel that this we could include regardless of `verbose` setting? Since the script takes a while, would be great to know that it has started.